apps/web/components/Organization/Wrapper/OrganizationWrapper.tsx

```diff
@@ -283,6 +290,15 @@ export const OrganizationFooter: React.FC<FooterProps> = ({
         <LandskjorstjornFooter footerItems={organization.footerItems} />
       )
       break
+    case 'rikislogmadur':
+    case 'office-of-the-attorney-general-civil-affairs':
+      OrganizationFooterComponent = (
+        <RikislogmadurFooter
+          title={organization.title}
+          footerItems={organization.footerItems}
+          logo={organization.logo?.url}
+        />
+      )
```

Comment:
Missing the `break` statement (which may cause issues in the future, when more cases are added).